ddtrace/internal/telemetry/writer.py

```diff
@@ -205,6 +235,7 @@ def __init__(self, is_periodic=True):
         self._debug = asbool(os.environ.get("DD_TELEMETRY_DEBUG", "false"))
 
         self._client = _TelemetryClient(self.ENDPOINT_V2)
+        self._module_watchdog = TelemetryWriterModuleWatchdog()
```

Comment:
`ModuleWatchdog`s are singletons so we can avoid storing a reference here. We can simply call the `install` class method when `enable` is called, and `uninstall` when `disabled` is called.

Reply:
Good advice, didn't notice it was a singleton.